docs/tutorials/getting_started/install_templates/idaes_data.md

```diff
@@ -0,0 +1,5 @@
+.. note::
+    If you want to install multiple versions of IDAES in different environments, you can set the ``IDAES_DATA`` environment variable before installing
+    or using IDAES.  The easiest way to do this is to set the environment variable as part of activating your environment before installing IDAES.  
+    Setting this environment variable will keep binary extensions and configuration files separate for the different IDAES versions.  For more 
+    information about setting environment variables as part of a conda or virtual environment, see the conda or venv documentation.  
```

Comment:
I think it'd be useful to add links to the respective pages. I've found the one for Conda on the documentation [here](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables), but I wasn't able to find the same for "vanilla" Python venvs.